spec/spec_helper.cr

```diff
@@ -1,6 +1,5 @@
 require "spec"
 require "../src/timecop"
 
-def times_effectively_equal(time1, time2, seconds_interval = 1.0)
-  (time1 - time2).abs.to_f <= seconds_interval
-end
+Spec.before_each { Timecop.return }
+Spec.after_each { Timecop.return }
```

Comment:
We probably don’t need this at both before and after

Reply:
I think we do in order to ensure a "clean-slate".

Reply:
This is not a problem with this pull, but it's redundant. Running the tests results in this:

- Timecop.return (before_each)
- Test 1
- Timecop.return (after_each)
- Timecop.return (before_each)
- Test 2
- Timecop.return (after_each)